projects/plugins/jetpack/extensions/shared/jetpack-plugin-sidebar.js

```diff
@@ -16,6 +17,19 @@ const { Fill, Slot } = createSlotFill( 'JetpackPluginSidebar' );
 
 export { Fill as default };
 
+/**
+ * Open Jetpack plugin sidebar by default when URL includes query param.
+ */
+function openJetpackSidebar() {
+	dispatch( 'core/interface' ).enableComplementaryArea(
+		'core/edit-post',
+		'jetpack-sidebar/jetpack'
+	);
+}
+const queryParams = new URLSearchParams( window.location.search );
+const jetpackSidebarIsOpen = queryParams.get( 'jetpackSidebarIsOpen' );
+jetpackSidebarIsOpen && openJetpackSidebar();
```

Comment:
we can move this call just below to the plugin registration?

Reply:
Also, probably we should check if its value is `true`, something like...
`jetpackSidebarIsOpen === 'true'`. Otherwise, it will execute the `openJetpackSidebar()` for any value of the `jetpackSidebarIsOpen` var in the query string.

Reply:
```suggestion
openJetpackSidebar();
```

Reply:
I was wondering about that, but I thought it would need to be a component in the `render` function, and [rendering an empty component ](https://github.com/Automattic/jetpack/pull/21156#discussion_r714185222) seemed weird. I can take another look at [registerPlugin()](https://github.com/WordPress/gutenberg/blob/3da717b8d0ac7d7821fc6d0475695ccf3ae2829f/packages/plugins/src/api/index.js#L115) or let me know if I missed something.